marimbabot_planning/README.md

```diff
@@ -1,15 +1,73 @@
 # TAMS Master Project 2022/2023 - Planning
 
 ### Test the motion in RViz / simulation environment
-Launch the Demo Mode as described in the main README.md. Then start the planning package with:
+Launch the Demo Mode as described in the main README.md, please run: 
 
 ```bash
-roslaunch marimbabot_planning marimbabot.launch
+roslaunch marimbabot_ur5_moveit_config demo.launch
```

Comment:
This is already mentioned in our README. And also, this is the single mallet one.